Lib/test/test_memoryview.py

```diff
@@ -564,25 +567,20 @@ def __bool__(self):
                 release()
                 return True
 
-        ba = None
         m = memoryview(bytearray(b'\xff'*size))
         with self.assertRaises(ValueError):
             m[MyIndex()]
 
-        ba = None
```

Comment:
It is necessary. The old bytearray should be deallocated before creating a new bytearray, so all bytearrays will be allocated at the same place.